src/go/configgenerator/listener_generator.go

```diff
@@ -237,6 +238,29 @@ func makeHttpConMgr(opts *options.ConfigGeneratorOptions, route *routepb.RouteCo
 		},
 		UseRemoteAddress:  &wrapperspb.BoolValue{Value: opts.EnvoyUseRemoteAddress},
 		XffNumTrustedHops: uint32(opts.EnvoyXffNumTrustedHops),
+		// Converting the error message for requests rejected by Envoy to JSON format:
+		//
+		//    {
+		//       "code": "http-status-code",
```

Comment:
In ESPv1, code is the canonical grpc status code, not a http status code. Unfortunately, this will be a breaking change.

I assume there is no way to easily fix this?

Reply:
Ack.  Yeap,  it is a breaking change.  But I don't understand why ESPv1 is sending grpc status code for http calls. I think it make more sense to send http status code.

Reply:
I see. I guess even ESPv1 doesn't follow the same error format that ESF / google APIs do: https://cloud.google.com/apis/design/errors#http_mapping

Seems fine, but we need to document in the migration guide.